docroot/themes/contrib/civictheme/civictheme_library/components/00-base/collapsible/collapsible.scss

```diff
@@ -18,9 +18,13 @@
       &::after {
         @include civictheme-icon-size('regular');
 
-        content: $civictheme-collapsible-icon;
+        content: $civictheme-mobile-collapsible-icon;
```

Comment:
Added second size icon - regular icon size is:
medium breakpoint and above: 24px
mobile breakpoint: 20px
But svg is 24px for all sizes causing misalignment - added second svg for mobile size at 20px.